build.sbt

```diff
@@ -175,6 +175,16 @@ lazy val commonSettings = instanceSettings ++ clearSourceAndResourceDirectories
   ),
   //maxErrors := 10,
   setIncOptions,
+  // http://stackoverflow.com/questions/16934488
+  apiMappings ++= {
+    Option(System.getProperty("sun.boot.class.path")).flatMap { classPath =>
```

Comment:
This returns `None` on JDK 11. Switching to JDK 11 as our main build/test/publish environment is eons off, so I don't think it makes the PR unmergeable. I see over at https://github.com/ThoughtWorksInc/sbt-api-mappings/pull/13/files they made it print a warning in this case. Perhaps it merits at least a comment?

Reply:
I figured a warning would just be ignored, but it could also double as a code comment, so  I'll add a warning.

Reply:
It would be cool if that if the API mappings were to stop working for a release, either for this snippet or for some unforeseen reason, but I wasn't sure how to guarantee they're working.